Tests/Clappr_Tests/Classes/Mocks/PlayerItemMock.swift

```diff
@@ -2,12 +2,21 @@ import AVFoundation
 
 class PlayerItemMock: AVPlayerItem {
     private var itemAccessLog: AVPlayerItemAccessLog
-
-    init(accessLogEvent: AccessLogEventMock) {
+    private var isFinished: Bool
+    private var _duration: CMTime
+    private var _currentTime: CMTime
```

Comment:
Can we use other name to this variable since it is not common on Swift Guidelines to use underscore in private variables?
https://www.natashatherobot.com/swift-magic-public-getter-private-setter/

Maybe currentTimeMocked or currentTimeToReturn.

Reply:
Done ;)